src/main/java/org/broadinstitute/dsde/consent/ontology/cloudstore/GCSHealthCheck.java

```diff
@@ -0,0 +1,28 @@
+package org.broadinstitute.dsde.consent.ontology.cloudstore;
+
+import com.codahale.metrics.health.HealthCheck;
+import com.google.api.client.http.HttpResponse;
+import org.apache.commons.io.IOUtils;
+
+public class GCSHealthCheck extends HealthCheck {
+
+    private GCSStore store;
+
+    public GCSHealthCheck(GCSStore store) {
+        this.store = store;
+    }
+
+    @Override
+    protected Result check() throws Exception {
+
+        // Attempt to read the root of the bucket; returns file list
```

Comment:
this could be a heavyweight health check if the bucket has many items (by default, GCS will use a limit of 1000 items, I believe). I understand that in practice, this bucket won't have a lot of items in it - but we should keep an eye on this for the future, and maybe switch to getting the bucket metadata instead of a list of the bucket contents.